docs/docsite/rst/command_guide/cheatsheet.rst

```diff
@@ -121,4 +121,25 @@ ansible-doc
 .. code-block:: bash
 
   ansible-doc -t module -l
-  #...
\ No newline at end of file
+  #...
+
+  ansible-config
+  ==============
+
+.. code-block:: bash
+
+*  The ansible-config utility allows users to see all the configuration settings available,
+   their defaults, how to set them and where their current value comes from.
+
+*  positional arguments:
+
+-v - causes Ansible to print more debug messages. 
+
+-c - path to configuration file, defaults to first file found in precedence.
+
+-f - Filter down to a specific plugin type.
+
+-t  - <TYPE>, --type  <TYPE> - Filter down to a specific plugin type.
+
+--format  <FORMAT>, -f  <FORMAT> - Output format for list,dump,int.
```

Comment:
I think it might be useful to give examples of the `list`, `dump`, `view`, `init` args here. The options are already available in docs here: https://docs.ansible.com/ansible/latest/cli/ansible-config.html#ansible-config

Reply:
Thanks @oraNod .
I've added the actions & their common options too in bullet form.